spiel/ut/spiel_ci_ut.c

```diff
@@ -544,18 +544,23 @@ void test_spiel_bc_stats(void)
 
 static void spiel_repair_start(const struct m0_fid *pool_fid,
 			       const struct m0_fid *svc_fid,
-			       enum m0_repreb_type  type)
+			       enum m0_repreb_type  type,
+                               bool isDtreb)
 {
 	struct m0_spiel_repreb_status *status;
 	enum m0_cm_status              state;
 	int                            rc;
 
 	rc = (type == M0_REPREB_TYPE_SNS) ?
-		m0_spiel_sns_repair_start(&spiel, pool_fid) :
```

Comment:
How about introducing a new function m0_spiel_sns_dtrebalance_start(..) to make it consistent with   m0_spiel_sns_repair_start() and m0_spiel_sns_rebalance_start()?